sigs-and-wgs/archive/sig-core/decisions/README.md

```diff
@@ -4,4 +4,4 @@
 
 This directory contains records of decisions made by the Core SIG and the Kyma project members before end of May 2019.
 
-Currently, decisions are made based on issues or pull requests that have the decision flag assigned. The decision-making process is described [here](https://github.com/kyma-project/community/blob/main/governance.md).
+Currently, decisions are made based on issues or pull requests that have the decision flag assigned. The decision-making process is described [here](https://github.com/kyma-project/community/blob/main/docs/governance/01-governance.md).
```

Comment:
Why not here: https://github.com/kyma-project/community/blob/main/docs/governance/01-governance.md#decision-making ?